test/SlowTests/SlowTests/StudioTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents.Indexes;
+using Raven.Server.Documents.Operations;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.SlowTests
+{
+    public class StudioTests : RavenTestBase
+    {
+        public StudioTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Theory]
+        [RavenData(DatabaseMode = RavenDatabaseMode.Single)]
+        public async Task CanGetStudioFooterStatistics(Options options)
+        {
+            using (var store = GetDocumentStore(options))
+            {
+                using (var session = store.OpenSession())
+                {
+                    session.Store(new User { Name = "SomeDoc" });
+                    session.SaveChanges();
+                }
+                await store.ExecuteIndexAsync(new AllowedUsers());
+                WaitForIndexing(store);
+
+                var stats = store.Maintenance.Send(new GetStudioFooterStatisticsOperation());
+
+                Assert.NotNull(stats);
+                Assert.Equal(1, stats.CountOfIndexes);
+                Assert.Equal(2, stats.CountOfDocuments);
+                Assert.Equal(0, stats.CountOfIndexingErrors);
+                Assert.Equal(0, stats.CountOfStaleIndexes);
+            }
+        }
+
+        [Theory]
+        [RavenData(DatabaseMode = RavenDatabaseMode.Sharded)]
+        public async Task ShardedCanGetStudioFooterStatistics(Options options)
+        {
+            //TODO: can unify with non sharded mode after all FooterStatistics fields are decided
+            using (var store = GetDocumentStore(options))
+            {
+                using (var session = store.OpenSession())
+                {
+                    session.Store(new User { Name = "SomeDoc" });
+                    session.SaveChanges();
+                }
+                await store.ExecuteIndexAsync(new AllowedUsers());
+                WaitForIndexing(store, sharded: true);
+
+                var stats = store.Maintenance.Send(new GetStudioFooterStatisticsOperation());
+
+                Assert.NotNull(stats);
+                Assert.Equal(1, stats.CountOfIndexes);
+                Assert.Equal(2, stats.CountOfDocuments);
+                Assert.Equal(null, stats.CountOfIndexingErrors);
```

Comment:
when you will rebase Options will have 2 additional properties. DatabaseMode and SearchEngineMode. Use them to do a proper assert and merge the test.

Reply:
done